torch/_ops.py

```diff
@@ -382,6 +382,9 @@ def wrapper():
     def __str__(self):
         return f"{self.name()}"
 
+    def __repr__(self):
+        return f"torch.ops._higher_order_ops.{self._name}"
```

Comment:
Why adding this? If it's about the name of the call_torchbind, we could do `call_torchbind.__module__ = "torch.ops.higher_order". We've done it for while_loop previously

Reply:
The error is when fx tries to compile the operator as python code:
```
  File "/data/users/angelayi/pytorch2/torch/fx/graph_module.py", line 724, in recompile
    cls.forward = _forward_from_src(self._code, python_code.globals, co_fields)
  File "/data/users/angelayi/pytorch2/torch/fx/graph_module.py", line 90, in _forward_from_src
    return _method_from_src(
  File "/data/users/angelayi/pytorch2/torch/fx/graph_module.py", line 100, in _method_from_src
    _exec_with_source(src, globals_copy, co_fields)
  File "/data/users/angelayi/pytorch2/torch/fx/graph_module.py", line 86, in _exec_with_source
    exec(compile(src, key, "exec"), globals)
  File "<eval_with_key>.0 from /data/users/angelayi/pytorch2/torch/fx/experimental/proxy_tensor.py:652 in wrapped", line 6
    with_effects = torch._higher_order_ops.effects.with_effects(arg0_1, <torch._ops.HigherOrderOperator object at 0x7f8edb7d4520>, _torchbind_obj0, 'add_tensor', arg1_1);  arg0_1 = _torchbind_obj0 = None
                                                                        ^
SyntaxError: invalid syntax
```

I tried adding `call_torchbind.__module__ = "torch.ops.higher_order"` but it still ran into the same issue :/

Reply:
nvm we figured out to fix in [_get_repr](https://github.com/pytorch/pytorch/blob/c63ccead5effbacfa40db14f11b63b16d3996aaf/torch/fx/graph.py#L445)